<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	border-radius:10px;
}</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLineEdit" name="usernameEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>237</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
}
QLineEdit:hover{
	border-color: rgb(47, 32, 255);
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>267</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
}
QLineEdit:hover{
	border-color: rgb(47, 32, 255);
}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRegister">
   <property name="geometry">
    <rect>
     <x>387</x>
     <y>239</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(3, 146, 255);
}
QPushuButton:hover{
	font: 9pt &quot;SimSun-ExtB&quot;;
	text-decoration: underline;
}</string>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonForgetPassword">
   <property name="geometry">
    <rect>
     <x>387</x>
     <y>269</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	
	color: rgb(3, 146, 255);
}</string>
   </property>
   <property name="text">
    <string>找回密码</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="rememberPsd">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoLogin">
   <property name="geometry">
    <rect>
     <x>281</x>
     <y>310</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>自动登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed{
	
	background-color: rgb(6, 118, 255);
}
QPushButton:hover{
	
	background-color: rgb(7, 181, 255);
}
QPushButton{
	
	background-color: rgb(30, 195, 255);
	border-radius:4px;
}</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFlicker">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>360</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="moreAccountLogin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="loginState">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>295</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="userHead">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
